runelite-client/src/main/java/net/runelite/client/plugins/itemstats/ItemStatChanges.java

```diff
@@ -214,6 +215,11 @@ private void init()
 		add(heal(HITPOINTS, 20), PADDLEFISH);
 		add(new GauntletPotion(), EGNIOL_POTION_1, EGNIOL_POTION_2, EGNIOL_POTION_3, EGNIOL_POTION_4);
 
+		// Soul Wars
+		add(new SoulWarsPotion(), POTION_OF_POWER1, POTION_OF_POWER2, POTION_OF_POWER3, POTION_OF_POWER4);
```

Comment:
Besides for the sake of brevity in this class, is there any reason to put the changes of the potion in its own class? afaict the stat changes can be put here as a `combo(6, ...)`